docs/DEVELOPMENT.md

```diff
@@ -2,6 +2,7 @@
 - make
 - golang 1.18.y
 - g++
+- yq (4.x.y)
```

Comment:
```suggestion
- yq (4.x.y, [the Go one](https://github.com/mikefarah/yq), not Python) 
```
The Python one is v3.x.x, but it is a common source of irritation, especially as the less popular python tool is often packaged as 'yq'.